### DISCUSSION TITLE
Error: Resource not accessible by integration

### DISCUSSION THREAD

**Discussion by FreaxMATE** [rossjrw/pr-preview-action#75] [Q&A] [answered]
I get the error in https://github.com/rossjrw/pr-preview-action/discussions/68 and https://github.com/rossjrw/pr-preview-action/discussions/59 however on a PR from the same repository and without pushing to a different repository.
My action:
``` yaml
# .github/workflows/preview.yml
name: Deploy PR previews

on:
  pull_request:
    types:
      - opened
      - reopened
      - synchronize
      - closed

permissions:
  contents: write
  pull-requests: write

jobs:
  deploy-preview:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: actions/setup-python@v4
        with:
          python-version: 3.x
      - run: echo "cache_id=$(date --utc '+%V')" >> $GITHUB_ENV
      - name: Install dependencies
      - uses: actions/cache@v3
        with:
          key: mkdocs-material-${{ env.cache_id }}
          path: .cache
          restore-keys: |
            mkdocs-material-
      - run: pip install mkdocs-material ipykernel nbconvert traitlets jupytext pandas python-markdown-math mkdocs-simple-hooks pillow cairosvg matplotlib scipy plotly ipywidgets
      - name: Build site
      - run: mkdocs build
      - name: Deploy preview
        uses: rossjrw/pr-preview-action@v1
        with:
          source-dir: ./site/
```

and this deployment: https://github.com/Hephaestus-Energy-Forge/Hephaestus-Energy-Forge/actions/runs/7597038400/job/20691926420

**Comment by rossjrw** [comment, marked as answer]
@FreaxMATE A couple of things here: 

1. It looks like the preview is going to the wrong place. It took me a good while to work out why your preview was being deployed to the `pr-preview` branch instead of to `gh-pages`, but it looks like that's because you started [the pull request](https://github.com/Hephaestus-Energy-Forge/Hephaestus-Energy-Forge/pull/2) _before_ you removed the branch parameter from the workflow file in this commit: https://github.com/Hephaestus-Energy-Forge/Hephaestus-Energy-Forge/commit/ff680cf49fcf5cecbda35dbedd6c94f52d678601. It looks like you then opened and closed the PR a lot to try to get it to update. To get it to update, you'll need to get those changes from the main branch into the PR branch - `git checkout feature/chapter-1` then either `git merge main` or `git rebase main`.


2. As for why the preview is failing, it's not - it's the addition of the sticky comment that's failing. As vorburger discovered in one of the threads you linked, it might help to add `permissions: issues: write` to your workflow file. It's not in the workflow that you pasted in this discussion, but I can see in your repo that you've actually already done that. The PR is still using the old version of the workflow file before all of your fixes. Updating it should resolve that.

TL;DR: You've already fixed the problem in your `main` branch but haven't updated `feature/chapter-1` with those fixes. Merge those changes into it, or rebase it, and the feature branch should be updated.

Ping me here if that doesn't work.

**Reply by FreaxMATE** [reply]
Thank you for looking into this! Yeah after updating the branch it worked.
